src/Transformer/EmojiStringTransformer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types = 1);
+
+namespace Diaclone\Transformer;
+
+use Diaclone\Exception\MalformedInputException;
+use Diaclone\Exception\TransformException;
+use Diaclone\Resource\ResourceInterface;
+use Diaclone\Traits\EmojiTrait;
+use Exception;
+
+class EmojiStringTransformer extends AbstractTransformer
```

Comment:
`EmojiStringTransformer` sounds like it's made for converting emojis but in reality it serves the purpose of converting all our textual fields. It also does html entity encoding/decoding. Something tells me we might add another string pre-processing step here in the future. In the future we might think about refactoring into a generic string transformer with options.

Reply:
how do you anticipate passing in params?
```
$transformers = [
   'title' => StringTransformer::class . ":uuid,emoji",
]
```